lib/portlayer/network/context.go

```diff
@@ -1412,6 +1412,71 @@ func (c *Context) deleteScope(s *Scope) {
 	delete(c.scopes, s.Name())
 }
 
+func (c *Context) UpdateContainerNameInScope(h *exec.Handle) error {
```

Comment:
Please add some comments about this api, and why it is related to ip conflict.

Reply:
Will do. thanks